app/src/main/res/layout/splash_screen.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/yellow_primary">
+
+    <ImageView
+        android:id="@+id/splash_screen_image"
+        android:layout_width="320dp"
```

Comment:
ini pake fixed size gini gpp ? ntar kalo deviceny kecil ato devicenya besar jd aneh gk ?

mungkin bisa pake rasio sm margin

Reply:
maybe you can ask @jeeshakira for this issue, beacuse it depends on her that we gonna use ratio or etc